layout/MDXLayoutPagination.tsx

```diff
@@ -29,8 +29,8 @@ export const MDXLayoutPagination = () => {
             >
               <Flex.Column align="start" gap={Spacing.S} sx={{ textAlign: 'left' }}>
                 <Flex.Row align="center" gap={Spacing.S}>
-                  <Icon.ArrowLeft size="12px" />
-                  <Text.S10>{translations.global.previous}</Text.S10>
+                  <Icon.ArrowLeft title="" size="12px" />
```

Comment:
is this empty text necessary

Reply:
Yes, because all icons have a default title, but in this case we don't want it to be read by screen readers. The button already has accessible text.

Reply:
This is the equivalent of doing `alt=""` on an image to mark it as decorative.